apps/wallet-mobile/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -72,8 +71,6 @@ export class ErrorBoundary extends Component<Props, State> {
 
               <Spacer height={16} />
             </View>
-
```

Comment:
this was removed by the vendor, when it shouldn't.

Reply:
done